packages/shared/src/apply/output.ts

```diff
@@ -54,10 +54,7 @@ export function applyOutputPlugin(api: SharedRsbuildPluginAPI) {
         const workerPath = getDistPath(config.output, 'worker');
         const filename = posix.join(workerPath, `[name].js`);
 
-        chain.output
-          .filename(filename)
-          .chunkFilename(filename)
-          .libraryTarget('commonjs2');
```

Comment:
This option is added to support Modern.js (EdenX) worker.

If we remove this, we should add this `libraryTarget` config to the Modern.js app-tools.